src/Libraries/Microsoft.Extensions.DependencyInjection.AutoActivation/AutoActivationExtensions.cs

```diff
@@ -276,6 +276,8 @@ private static IServiceCollection Activate(this IServiceCollection services, [Dy
         return services;
     }
 
+    // exclude from coverage until https://github.com/microsoft/codecoverage/issues/38 is addressed
+    [ExcludeFromCodeCoverage]
     private static IServiceCollection AddHostedServiceIfNotExist(this IServiceCollection services)
     {
 #if NETFRAMEWORK
```

Comment:
Do we build for .NET FX?

Reply:
Yes, all libraries are built for `.net8;.netcore3.1;net462` unless explicitly opted-out
https://github.com/dotnet/extensions/blob/5352825bd11925eff0cc0dcd3283e1481fa75cb6/src/Libraries/Directory.Build.props#L5
https://github.com/dotnet/extensions/blob/5352825bd11925eff0cc0dcd3283e1481fa75cb6/Directory.Build.props#L12-L31